From 8d006144c27a6d4240b81c7682c20512e4cd62c8 Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Wed, 10 Apr 2024 14:32:49 +0300
Subject: [PATCH] [SW-180536] gaudi3: tpm: remove pcr index get function

For Gaudi3, instead of separately extending PCRs
for individual ppboot and preboot images, we will
consolidate the process by extending the PCR of
the manifest.

In preparation for this change, this patch removes
the functions to fetch the PCR index for ppboot and
preboot images. Weak functions will be used in place
of removed ones, since they return the same values.

Change-Id: I666715757ee376992e03cd7bc8aea977af465bb5
Signed-off-by: Varsha Maira <vmaira@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 763129f..29478e8 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -1470,16 +1470,6 @@
 	return count;
 }
 
-uint8_t get_ppboot_pcr_index(void)
-{
-	return TPM2_PPBOOT_PCR_IDX;
-}
-
-uint8_t get_preboot_pcr_index(void)
-{
-	return TPM2_PREBOOT_PCR_IDX;
-}
-
 static bool get_ppboot_img_desc(struct boot_desc *desc)
 {
 	u32 idx;
